doc/Interns/MyFirstCommit.md

```diff
@@ -1,5 +1,5 @@
 # PowerToys - My First Commit #
-
+Date: May 6th 2020 Time: 1:43 Eastern
```

Comment:
This is correct, just in the wrong file.  Please review the instructions.  Undo this change, and update the correct new file.